src/routes/auth.route.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const { authController } = require('../controllers/auth.controller');
+const { authMiddleware } = require('../middlewares/authMiddleware');
+
+const router = express.Router();
+
+router.post('/signup', authController.authValidation);
+router.post('/signup', authController.signUp);
```

Comment:
The `authValidation` middleware is applied twice for the `/signup` route. Consider combining these into a single route definition to avoid redundancy.